Header header
string autoware_state
int32 control_mode
int32 gate_mode
bool emergency_stopped
string stop_reason
diagnostic_msgs/DiagnosticStatus[] diagnostics
bool autonomous_overriden
